scripts/contentScript.js

```diff
@@ -380,6 +380,99 @@ async function FilesInPrBitbucket(response) {
 	});
 }
 
+const githubHunkHighlight = async (apiResponses) => {
+
+	// Todo : optomization needed to not highlight next deleted line space if not present in response.
+	let getFileName = Array.from(document.querySelectorAll('div[data-tagsearch-path]'));
+	getFileName.forEach(async (item) => {
+		let FileContent = item.getAttribute('data-tagsearch-path');
```

Comment:
```suggestion
		const fileContent = item.getAttribute('data-tagsearch-path');
```

1. use `const` wherever possible
2. We use camelCase for normal variables. PascalCase is used for classes, singletons etc.